dist/rules/no-internal.js

```diff
@@ -90,7 +91,11 @@ module.exports = {
       if (!declaration)
         return false;
       const fileName = getFileName(declaration.parent);
-      const inCheckedPackage = checkedPackageRegexes.some((r) => r.test(fileName));
+      const packageSegments = fileName.split("node_modules" + path.sep);
```

Comment:
now I'm curious... does eslint really give you file names with backslashes on Windows? I guess in that case `^@bentley/` and `^@itwin/` should also use `path.sep`.

Reply:
I have no idea, I'm on ubuntu. I was just being safe. But you're right.  I need to use an escaped backslash on windows.

Reply:
done